db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2024_01_18_000000) do
+ActiveRecord::Schema[7.2].define(version: 2024_01_18_000000) do
```

Comment:
@emmaachesongray Please check this change with Martyn before it goes out

Reply:
this change is fine to keep in 👍